platform/pipelines/jobs-platform-deployment.yaml

```diff
@@ -54,6 +54,17 @@ jobs:
               displayName: 'Run platform search app'
               workingDirectory: '${{ parameters.workspaceDir }}/search-app'
               failOnStderr: true
+              
+            - task: ExtractFiles@1
+              displayName: 'Extract platform database'
+              inputs:
+                archiveFilePatterns: '${{ parameters.workspaceDir }}/platform-db/Platform.Database.zip'
+                destinationFolder: '${{ parameters.workspaceDir }}/db'
+          
+#            - script: ./Platform.Database.exe "$(platform-db-connection-string)"
```

Comment:
has this been commented out on purpose?

Reply:
Because there is no infrastructure, connection string, scripts, etc to run